guides/common/modules/proc_upgrading-hosts-to-next-major-release.adoc

```diff
@@ -6,6 +6,7 @@ Below upgrade paths are possible:
 
 * {EL} 7 to {EL} 8
 * {EL} 8 to {EL} 9
+* {EL} 9 to {EL} 10
 
 .Prerequisites
 * Ensure that your {EL} hosts meet the requirements for the upgrade.
```

Comment:
Below we list the various OS prerequisites. I'd expect we don't merge this until we have a link to the official docs with those prerequisites.

Reply:
Thanks!